en/prototyping/usability-study-results-overview.md

```diff
@@ -137,8 +108,24 @@ Watch some of the videos we have on usability studies.
 * [Take part in a usability study](https://www.youtube.com/watch?v=dfrxhBW5NFQ?rel=0&autoplay=1)
 * [View usability study results](https://www.youtube.com/watch?v=00P6DwMbYMM?rel=0&autoplay=1)
 
+
 [1]: https://www.youtube.com/watch?v=vilyDL4fDT0
+[2]: https://cloud.indigo.design/api/shares/qv6uzwx9jwu3/files/project/Money%20App.zip
+[3]: https://cloud.indigo.design
+
+[a-1]: #Test-analytics-overview
+[a-2]: #Task-completion-funnel-and-1st-click-report
+[a-3]: #User-segments-and-participation-details
+[a-4]: #Explore-click-path
+[a-5]: #Video-replay-of-user-sessions
+
+[topic-1]: setting-up-a-usability-study.md
+
+[img-1]: ..images/understanding_the_usability_test_report_1.png
+[img-2]: ..images/understanding_the_usability_test_report_2.png
+[img-3]: ..images/understanding_the_usability_test_report_3.png
+[img-4]: ..images/understanding_the_usability_test_report_4.png
+[img-5]: ..images/understanding_the_usability_test_report_5.png
+[img-6]: ..images/understanding_the_usability_test_report_6.png
+[img-7]: ..images/understanding_the_usability_test_report_7.png
```

Comment:
You also need to fix the images here

Reply:
I updated how I reference images now @StefanIvanov. All images should be showing now.